src/Tickets/Commerce/Order_Modifiers/Repositories/Order_Modifiers.php

```diff
@@ -185,46 +188,45 @@ public function find_by_id( int $id ): Order_Modifier {
 	 * Search for Order Modifiers based on the given criteria.
 	 *
 	 * @param array $args          {
-	 *                             Optional. Arguments to filter the query.
-	 *
-	 *     @type string $search_term The term to search for (e.g., in display_name or slug).
-	 *     @type string $orderby     Column to order by. Default 'display_name'.
-	 *     @type string $order       Sorting order. Either 'asc' or 'desc'. Default 'asc'.
+	 *     Optional. Arguments to filter the query. See get_default_query_params() method for the full list of parameters.
+	 *
+	 *     @type string   $search_term The term to search for (e.g., in display_name or slug).
+	 *     @type string   $orderby     Column to order by. Default 'display_name'.
+	 *     @type string   $order       Sorting order. Either 'asc' or 'desc'. Default 'asc'.
+	 *     @type int      $limit       The number of results to return. Default 10. Using -1 disables the limit.
+	 *     @type int      $page        The page number to retrieve. Default 1.
+	 *     @type string[] $status      The status of the modifiers to filter by. Default 'active'.
 	 * }
 	 *
 	 * @return Order_Modifier[] An array of Order_Modifiers or an empty array if none found.
 	 */
 	public function search_modifiers( array $args = [] ): array {
-		// Define default arguments.
-		$defaults = [
-			'search_term' => '',
-			'orderby'     => 'display_name',
-			'order'       => 'asc',
-		];
-
 		// Merge passed arguments with defaults.
-		$args = array_merge( $defaults, $args );
+		$args       = wp_parse_args( $args, $this->get_default_query_params() );
+		$valid_args = $this->get_valid_params( $args );
```

Comment:
Not huge, but better to use one term - `params` or `args` (in this case...`args`?)